hb_ucdn_get_unicode_funcs is referenced but not compiled

v1.4.1, Windows, MSVC, compiling with glib/gobject support.

```nmake /f Makefile.vc FREETYPE=1 GOBJECT=1```

fails at link time with

```
hb-unicode.obj : error LNK2001: unresolved external symbol _hb_ucdn_get_unicode_funcs
```

`hb-ucdn.cc` is not getting compiled, so the error is justified.

Probably introduced by 53543f8b3e19456279c86f711feba42ce976668c

[Currently `HB_UCDN_sources` is only included in non-glib builds.](https://github.com/behdad/harfbuzz/blob/af596a5655ee2ed09da17a7de79f619e1547dcd9/win32/config-msvc.mak#L199) Perhaps you want to include it for all builds now.

--- a/win32/config-msvc.mak
+++ b/win32/config-msvc.mak
@@ -184,9 +184,10 @@ HB_CFLAGS =	$(HB_CFLAGS) /DU_HAVE_INT8_T
 HB_SOURCES = $(HB_SOURCES) $(HB_ICU_sources)
 HB_HEADERS = $(HB_HEADERS) $(HB_ICU_headers)
 HB_DEP_LIBS = $(HB_DEP_LIBS) $(HB_ICU_DEP_LIBS)
-!else
-# If there is no GLib support, use the built-in UCDN
-# and define some of the macros in GLib's msvc_recommended_pragmas.h
+!endif
+
+!if "$(UCDN)" != "0"
+# Define some of the macros in GLib's msvc_recommended_pragmas.h
 # to reduce some unneeded build-time warnings
 HB_DEFINES = $(HB_DEFINES) /DHAVE_UCDN=1
 HB_CFLAGS =	\

